openstates/in/bills.py

```diff
@@ -42,201 +42,214 @@ class INBillScraper(BillScraper):
+                chamber = "lower" if a["chamber"]["name"].lower() == "house" else "upper"
+                date = a["date"]
+                date = datetime.datetime.strptime(date,"%Y-%m-%dT%H:%M:%S")
+                
+                action_type = []
+                d = action_desc.lower()
+                committee = None
+
+                reading = False
+                if "first reading" in d:
+                    action_type.append("bill:reading:1")
+                    reading = True
+
+                if ("second reading" in d
+                    or "reread second time" in d):
+                    action_type.append("bill:reading:2")
+                    reading = True
+
+                if ("third reading" in d
+                    or "reread third time" in d):
+                    action_type.append("bill:reading:3")
+                    if "passed" in d:
+                        action_type.append("bill:passed")
+                    if "failed" in d:
+                        action_type.append("bill:failed")
+                    reading = True
+
+                if "adopted voice vote" in d and reading:
+                    action_type.append("bill:passed")
+
+                if ("referred" in d and "committee on" in d
+                    or "reassigned" in d and "committee on" in d):
+                    committee = d.split("committee on")[-1].strip()
+                    action_type.append("committee:referred")
+
+
+                if "committee report" in d:
+                    if "pass" in d:
+                        action_type.append("committee:passed")
+                    if "fail" in d:
+                        action_type.append("committee:failed")
+
+                if "amendment" in d:
+                    if "pass" in d or "prevail" in d or "adopted" in d:
+                        action_type.append("amendment:passed")
+                    if "fail" or "out of order" in d:
+                        action_type.append("amendment:failed")
+                    if "withdraw" in d:
+                        action_type.append("amendment:withdrawn")
+
+
+                if "signed by the governor" in d:
+                    action_type.append("governor:signed")
+
+
+                if ("not substituted for majority report" in d
+                    or "returned to the house" in d
+                    or "referred to the senate" in d
+                    or "referred to the house" in d
+                    or "technical corrections" in d
+                    or "signed by the president" in d
+                    or "signed by the speaker"
+                    or "authored" in d
+                    or "sponsor" in d
+                    or "coauthor" in d
+                    or ("rule" in d and "suspended" in d)
+                    or "removed as author" in d
+                    or ("added as" in d and "author" in d)
+                    or "public law" in d):
+                        action_type.append("other")
```

Comment:
Do we need to add `"other"` to `action`s that already have a `type`? I would argue not, but I could see the other side of the argument.